python/api/api_views.py

```diff
@@ -73,6 +73,15 @@ def select_serializer(self, request):
         return self.serializer_class
 
 
+def getlist_from_querydict(query_params, field_name):
+    params = query_params.get(field_name, [])
+    if params:
+        fields = [param.strip() for param in params.split(',')]
+    else:
+        return params
+    return fields
+
```

Comment:
Readability is not very good. I spent 5 munites to read this code.
Python way would be this:

```
if not params:
   return []
return [param.strip() for param in params.split(',')]
```

Reply:
Ok